gcp/modules/gcp-secret-mgmt/main.tf

```diff
@@ -20,6 +20,7 @@ module "gcp-secret-mgmt" {
   encryption_keys    = "${var.encryption_keys}"
   storage_location   = "${var.infra_region}"
   keyring_name       = "${var.keyring_name}"
```

Comment:
Would it make sense to include the region in the name of the keyring? `"keyring-${var.infra_region}" maybe? This should eventually allow multiple keyrings in different regions to co-exist.

Reply:
It turns out that Keyrings in different regions need not have unique names. My dev environment has four Keyrings in different regions, all named "keyring".

Reply:
ok then, I did not know that :), thanks for info